ansible/bootstrap.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Bootstrap node servers
+  hosts: all
+  remote_user: master
+  become: yes
+
+  vars:
+    bootstrap_version: master
+    vault_role: ae-node
+    package: "https://s3.eu-central-1.amazonaws.com/aeternity-node-builds/aeternity-latest-ubuntu-x86_64.tar.gz"
```

Comment:
I prefer to have the `latest` stable package by default instead of latest unknown build.